### PR TITLE
Adding Panther.Audit to the Greynoise LUTs

### PR DESCRIPTION
**Pull request by nkulig**
### Background

The audit logs for Panther Console was left out of the mappings for the Greynoise lookup tables. This PR adds them in!

### Changes

Updated the Greynoise .yml files with the appropriate metadata.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (edyesed) [inline .vscode/settings.json:1]: This file should not be part of the PR.

**Review by edyesed — CHANGES_REQUESTED** [review]
Good To Go after removing .vscode/settings.json

File: .vscode/settings.json (hunk 1 above)
Comment (nkulig) [reply .vscode/settings.json]: Sorry about that. Fixed!

**Review by edyesed — APPROVED** [review]
adds greynoise enrichment to panther audit logtype
